hud-data/berryhud.json

```diff
@@ -1,12 +1,57 @@
 {
   "name": "BerryHUD",
-  "author": "hexereii",
+  "author": "heks",
   "social": {
-    "album": "https://imgur.com/a/ldt9NXv"
+    "steam_profile": "76561198041335138",
+    "steam_group": "",
```

Comment:
I believe @mastercoms prefers empty lines like this to just be deleted, so you can just remove line 6